Report the active APN in the property table (#87)
nerves-networking/vintage_net_qmi: README.md

| `channel`     | integer        | An integer that indicates the channel that's in use |
| `manufacturer` | string        | The name of the manufacturer of the modem |
| `model`       | string         | The name of the model of the modem        |

> | `apn`         | string         | The APN the modem is connected to |

Reviewer: ```suggestion
| `apn`         | string         | The APN that VintageNetQMI configured the modem to use |
```
